docs/configuration/cluster-misconfigurations.rst

```diff
@@ -0,0 +1,118 @@
+Kubernetes Misconfigurations (Popeye)
+************************************************
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes clusters and reports potential issues with resources and configurations.
+By default, every instance of Robusta that's connected to the UI will run a Popeye scan on startup. Further Popeye scans can be triggered in the UI, and all scans can be viewed there.
+
+With or without the UI, you can configure additional scans on a :ref:`schedule <Scheduled>` as shown below.
```

Comment:
First sentence on this page is good. (Popeye is a utility....) Can you change the sentences after that to start from more of an overview (you can integrate popeye with robusta etc) then go into details. E.g.

> `Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes clusters and reports potential issues with resources and configurations.
>
> By optionally integrating Popeye with Robusta you can: 
> 1. Get weekly Popeye scans in Slack via Robusta OSS (disabled by default, see below to configure)
> 2. View Popeye scans from all your clusters in the Robusta UI (enabled by default for UI users)